frontend/website-redesign/src/pages/Tech.re

```diff
@@ -185,7 +184,7 @@ module HowMinaWorks = {
           </b>
           {React.string(
              {js|
-            When anyone can enforce the rules by validating an irrevocable public
+            WheknowledgeBaseCategoryn anyone can enforce the rules by validating an irrevocable public
```

Comment:
Looks like this inadvertently slipped in

Reply:
Great catch, thanks.